boards/arm/nrf52840_pca10056/doc/nrf52840_pca10056.rst

```diff
@@ -167,6 +167,54 @@ You can build and flash the examples to make sure Zephyr is running correctly on
 your board. The button and LED definitions can be found in :file:
 `boards/arm/nrf52840_pca10056/nrf52840_pca10056.dts`.
 
+Using UART1
+***********
+
+The following approach can be used when an application needs to use
+more than one UART for connecting peripheral devices:
+
+1. Add device tree overlay file to the main directory of your application
+
+.. code-block:: console
+
+ $ cat nrf52840_pca10056.overlay
+  &uart1 {
+    compatible = "nordic,nrf-uarte";
+    current-speed = <115200>;
+    status = "ok";
+    tx-pin = <14>;
+    rx-pin = <16>;
+  };
+
+In the overlay file above, pin P0.16 is used for RX and P0.14 is used for TX
+
+2. Go to menuconfig and enable :option: `UART_1_NRF_UARTE`:
+   (top menu) -> Device Drivers -> Serial Drivers -> nRF UART nrfx drivers
+
+3. Use the UART1 as device_get_binding("UART_1")
+
+Overlay file naming
+===================
+The file has to be named ``<board>.overlay`` and placed in the app main directory to be
+picked up automatically by the device tree compiler.
+
+Selecting the pins
+==================
+To select the pin numbers for tx-pin and rx-pin:
+
+.. code-block::.. console
+	tx-pin = <pin_no>
+
+Open the `nRF52840 Product Specification`_, chapter 7 'Hardware and Layout'.
+In the table 7.1.1 'aQFN73 ball assignments' select the pins marked
+'General purpose I/O'.  Note that pins marked as 'low frequency I/O only' can only be used
+in under-10KHz applications. They are not sutiable for 115200 speed of UART.
+
+Translating 'Pin' into number for Device tree using following formula:
```

Comment:
Change to (note the ``::`` at the end of this first line, and the indent and blank lines):
```
Translate 'Pin' into number for Device tree by using the following formula::

   pin_no = b\*32 + a

where ``a`` and ``b`` are from the Pin value in the table (Pb.a).
For example, for P0.1, ``pin_no = 1`` and for P1.0, ``pin_no = 32``.
```